cpp/modmesh/toggle/RadixTree.hpp

```diff
@@ -41,29 +41,29 @@ namespace modmesh
 /**
  * Simple Timed Entry
  */
-class TimedEntry
-{
-public:
-    size_t count() const { return m_count; }
-    double time() const { return m_time; }
-
-    TimedEntry & add_time(double time)
-    {
-        ++m_count;
-        m_time += time;
-        return *this;
-    }
-
-    friend std::ostream & operator<<(std::ostream & os, const TimedEntry & entry)
-    {
-        os << "Count: " << entry.count() << " - Time: " << entry.time();
-        return os;
-    }
-
-private:
-    size_t m_count = 0;
-    double m_time = 0.0;
-}; /* end class TimedEntry */
+// class TimedEntry
+// {
+// public:
+//     size_t count() const { return m_count; }
+//     double time() const { return m_time; }
+
+//     TimedEntry & add_time(double time)
+//     {
+//         ++m_count;
+//         m_time += time;
+//         return *this;
+//     }
+
+//     friend std::ostream & operator<<(std::ostream & os, const TimedEntry & entry)
+//     {
+//         os << "Count: " << entry.count() << " - Time: " << entry.time();
+//         return os;
+//     }
+
+// private:
+//     size_t m_count = 0;
+//     double m_time = 0.0;
+// }; /* end class TimedEntry */
```

Comment:
Nitpick: Since we use version control, it's generally not necessary to comment out code. This practice can lead to confusion when reviewing historical commits. It might be more effective to remove obsolete code outright.